deploy/helm/trickster/templates/deployment.yaml

```diff
@@ -58,6 +57,8 @@ spec:
           {{- end }}
         {{- end }}
           volumeMounts:
+            - name: cfg-volume
```

Comment:
This looks suspect. If the user has several filesystem caches defined, the iterator below should populate them, so I am not sure if one should also be hardcoded?  Should we

Reply:
I think this just got removed by accident. Those filesystem caches 

https://github.com/Comcast/trickster/blob/v0.1.10/deploy/helm/trickster/templates/deployment.yaml#L57

This is actually pretty confusing. What is going on in this file...

If the path is `""`, which shouldn't happen ideally, we may have a bunch of volumes mounting over each other? I also don't see where the matching config volume is generated that this maps with is even generated.

Reply:
It is also kind of funny that unless you mount a pvc/pv in there the "filesystem" caches will just be in the docker "filesystem" which is just node memory...

Reply:
Yeah, having multiple of the same path try to mount over each other doesn't even work. D:

Reply:
It looks like it used to be that if your cache was a file system we would, and still do, a volume named `cache-volume`, then provide that in the volume mounts.

https://github.com/Comcast/trickster/pull/189/files#diff-9fa4494b678a0fd83a220b037f2b3e8aR60

That is no longer a thing.

Reply:
James and I have discussed offline what my plan was to rectify these discrepancies. 

The TL;DR is to rework how volumes are mapped given we support multiple caches now instead of one.

I'll leave a more details comment.